help/en/html/tools/Sensors.shtml

```diff
@@ -53,42 +51,16 @@
         "../apps/PanelPro/index.shtml">PanelPro pages</a> for
         more on Panels).</p>
 
-        <p>The change takes effect immediately, unless a debounce
-        timer is configured (see below), but only lasts until the
-        next time you change it in the program, or the next time
-        the input hardware on the layout changes the value.</p>
+        <p>The change takes effect immediately, unless <a href="#Debounce"> a debounce timer</a> is configured,
+        but only lasts until the next time you change it in the program, or the next time the input hardware on
+        the layout changes the value.</p>
 
-        <h2>Sensor Properties</h2>
-
-        <h3><a name="Debounce" id="Debounce"></a>Sensor
```

Comment:
This is the Sensor page, so discussing the debounce facility here seems appropriate. Is there a reason it needed to be removed?